letsencrypt/plugins/util_test.py

```diff
@@ -98,6 +98,11 @@ def test_listening_ipv6(self, mock_get_utility, mock_process, mock_net):
         self.assertEqual(mock_get_utility.call_count, 1)
         mock_process.assert_called_once_with(4420)
 
+    @mock.patch("letsencrypt.plugins.util.psutil.net_connections")
+    def test_access_denied_exception(self, mock_net):
+        mock_net.return_value = False
```

Comment:
I'm not sure what you mean. If the side effect raises exception, return will never be reached. So whatever you set to `mock_net.return_value` doesn't matter at all.